packages/config/src/rollup.config.mjs

```diff
@@ -40,29 +29,18 @@ const createConfig = (name, input, output) => ({
   ]
 });
 
-const mainConfig = createConfig('formkit', './index.js', 'dist');
-const componentConfigs = getComponentEntryPoints().map(({ name, input, output }) =>
-  createConfig(name, input, output)
-);
+const mainConfig = createConfig('formkit', './src/index.js', 'dist');
```

Comment:
Do we need this anymore? I don't think this file exists in the monorepo?

Reply:
this config runs under each component, `src/index.js` still exists for each components